### DISCUSSION TITLE
Zabbix SAML SLO(SingleLogoutService) Error

### DISCUSSION THREAD

**Discussion by iam-Justin** [zitadel/zitadel#7406] [Q&A]
Login is working. But when logout it report: SAML LogoutRequest/LogoutResponse not found. Only supported HTTP_REDIRECT Binding

Zabbix Conf:
```
[idp_entityid] => http://192.168.56.1:8080/saml/v2/metadata
[sso_url] => http://192.168.56.1:8080/saml/v2/SSO
[slo_url] => http://192.168.56.1:8080/saml/v2/SLO
[username_attribute] => displayName
[sp_entityid] => zabbix
```

Zitadel Conf:
```
<?xml version="1.0"?>
<md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" xmlns:ds="http://www.w3.org/2000/09/xmldsig#" entityID="zabbix">
    <ds:KeyInfo>
        <ds:X509Data>
            <ds:X509Certificate>
xxxxxxxx
            </ds:X509Certificate>
        </ds:X509Data>
    </ds:KeyInfo>
    <md:SPSSODescriptor protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol">
        <md:NameIDFormat>urn:oasis:names:tc:SAML:2.0:nameid-format:persistent</md:NameIDFormat>
        <md:AssertionConsumerService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="http://192.168.56.1:8081/index_sso.php?acs" index="0"/>
        <md:SingleLogoutService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect" Location="http://192.168.56.1:8081/index_sso.php?sls"/>
    </md:SPSSODescriptor>
</md:EntityDescriptor>
```

Where might the problem be?

**Comment by hifabienne** [comment]
@stebenz @livio-a @muhlemmer Can you have a look at this?

**Comment by stebenz** [comment]
@iam-Justin Maybe to get a bit more context:
- From where do you get the error?
- If it is from ZITADEL, what error code are you getting?
- Could you provide us with an example of the SAMLRequest for the logout?
- Do you have an application created in ZITADEL with the exact entitiyID?

**Comment by HimbeersaftLP** [comment]
@stebenz The problem appears to be that Zitadel ignores the `urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect` Binding of the `SingleLogoutService` and always uses `urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST`, but the [php-saml](https://github.com/SAML-Toolkits/php-saml) library that Zabbix uses doesn't support HTTP_POST for SLO.
This might be related: https://github.com/zitadel/zitadel/issues/10182

**Reply by livio-a** [reply]
I just made a comment on the mentioned issue: https://github.com/zitadel/zitadel/issues/10182#issuecomment-3295457756
Looking at the metadata, shouldn't the SLO endpoint be part of the `IDPSSODescriptor` or am I missing something?
